devdox/tests/live/app/test_main.py

```
@@ -0,0 +1,97 @@
+       - trigger shutdown of the client
+       - re-raise the ping exception
+
+    This uses the real driver and real network failure, no mocks.
+    """
+    cfg = MongoConfig(
+	    HOST="localhost",
+	    PORT=30000,
+	    DB="some_random_db",
+    )
+
+    with pytest.raises(Exception):
```

Ruff (0.14.7)

Do not assert blind exception: `Exception`

(B017)